pkg/middleware/request_metrics.go

```diff
@@ -120,13 +140,15 @@ func RequestMetrics(features featuremgmt.FeatureToggles, cfg *setting.Cfg, promR
 				// Need to type-convert the Observer to an
 				// ExemplarObserver. This will always work for a
 				// HistogramVec.
-				histogram.(prometheus.ExemplarObserver).ObserveWithExemplar(
+				durationHistogram.(prometheus.ExemplarObserver).ObserveWithExemplar(
 					elapsedTime, prometheus.Labels{"traceID": traceID},
 				)
 			} else {
-				histogram.Observe(elapsedTime)
+				durationHistogram.Observe(elapsedTime)
 			}
 
+			sizeHistogram.Observe(float64(rw.Size()))
```

Comment:
why not also observe with an exemplar for this one? seems like it might be useful...

Reply:
ngl the code kinda scared me a little bit so i just ignored it, figuring if it's the right thing to do someone will tell me!